static/app/components/modals/inviteMembersModal/index.spec.jsx

```diff
@@ -40,88 +43,69 @@ describe('InviteMembersModal', function () {
   });
 
   it('renders', function () {
-    const wrapper = mountWithTheme(
-      <InviteMembersModal {...modalProps} organization={org} />
-    );
+    render(<InviteMembersModal {...modalProps} organization={org} />);
```

Comment:
aha! this is not yet in `ts` and that is why the `modalProps` mock still works here

Reply:
yeah. What we should do in the future, is export `commonModalProps` or `makeModalProps` from our test utils that can set them up for you